.oxlintrc.json

```diff
@@ -8,10 +12,15 @@
 	"rules": {
 		"max-lines-per-function": "off",
 		"max-lines": "off",
-		"max-depth": "off"
+		"max-depth": "off",
+		"eqeqeq": ["error", "always", { "null": "ignore" }]
```

Comment:
Strict equality generally makes sense, but when comparing `undefined` and `null` it shouldn't be enforced, as they are often used interchangeably.

The idea of strict equality generally comes from JS quirks like `12 == "12" being true and so on...